packages/mobile/src/invite/reducer.ts

```diff
@@ -1,88 +1,32 @@
-import { Actions as AccountActions, ActionTypes as AccountActionTypes } from 'src/account/actions'
 import { Actions, ActionTypes, InviteDetails } from 'src/invite/actions'
 import { getRehydratePayload, REHYDRATE, RehydrateAction } from 'src/redux/persist-helper'
 import { RootState } from 'src/redux/reducers'
 
 export interface State {
-  isSendingInvite: boolean
-  isRedeemingInvite: boolean
   invitees: InviteDetails[]
-  redeemComplete: boolean
-  redeemedTempAccountPrivateKey: string | null
 }
 
 export const initialState: State = {
-  isSendingInvite: false,
-  isRedeemingInvite: false,
   invitees: [],
-  redeemComplete: false,
-  redeemedTempAccountPrivateKey: null,
 }
```

Comment:
We should add a migration to clean up the deleted keys.

Reply:
Is a full migration necessary in this case? I changed the handling of the `REHYDRATE` action to ignore any field other than `invitees`.  This field is a bit legacy and the whole reducer should probably go away entirely when we do the improvements to the invite experience, so this feels like it's enough for now, wdyt?

Reply:
Yep that's good!